openrtb_ext/imp_limelightDigital.go

```diff
@@ -0,0 +1,8 @@
+package openrtb_ext
+
+import "encoding/json"
+
+type ImpExtLimelightDigital struct {
```

Comment:
The [doc PR](https://github.com/prebid/prebid.github.io/pull/4297/files) has required fields adUnitId and adUnitType which are missing here. If those fields are required, could you please consider adding those here as well, if not, then please update the doc PR to match the parameters.

Reply:
@Sonali-More-Xandr It is required fields for our Prebid.js dapter. As noted in the docs PR https://github.com/prebid/prebid.github.io/pull/4297, we need to support the same set of parameters with our Prebid.js adapter.

Reply:
We can not remove these fields from doc because are required fields for our Prebid.js adapter. And we don't need to use these filed in our Prebid Server adapter. Should we add parameters that we don't use to our Prebid Server adapter?

Reply:
The server should be able to accept the same parameters, ignoring them if not needed server-side.

This is to allow publishers to use the server-to-server test module, which uses the same set of configuration for your bidder.

Reply:
@bretg Thanks, you are absolutely right. The server is already able to accept the same parameters, it ignore them. I added tests with this case in https://github.com/prebid/prebid-server/pull/2539/commits/0b780e88c6cda75ce2fa382633a7959061224e4b commit